src/components/EventCard/EventCard.tsx

```diff
@@ -0,0 +1,309 @@
+} from '@phosphor-icons/react';
+import getCountryFlag from '../../utils/getCountryFlag';
+import { BREAKPOINT_MD_QUERY } from '../../constants/breakpoints';
+import StrapiEvent, { EventType } from '../../models/strapi/StrapiEvent';
+import { IntlContext } from '../ContextProvider';
+import strapiMediaUrl from '../../utils/strapiMediaUrl';
+
+export interface EventCardProps {
+  event: StrapiEvent;
+}
+
+const MAX_LENGTH = 120;
+const LOCATION_MAX_LENGTH = 28;
+
+const getEventIcon = (eventType: string): JSX.Element => {
+  switch (eventType) {
+    case EventType.WEBINAR:
+      return <Webcam size={12} />;
+    case EventType.CONFERENCE:
+      return <Headset size={12} />;
+    case EventType.MEET_UP:
+      return <UsersThree size={12} />;
+    case EventType.FOREST_WALK:
+      return <PersonSimpleWalk size={12} />;
+    case EventType.PARTNER_EVENT:
+      return <Handshake size={12} />;
+    case EventType.LUNCH_AND_LEARN:
+      return <BowlFood size={12} />;
+    case EventType.FAIR:
+      return <ChalkboardTeacher size={12} />;
+    case EventType.FESTIVAL:
+      return <Confetti size={12} />;
+    case EventType.ROADSHOW:
+      return <ProjectorScreenChart size={12} />;
+    default:
+      return <Info size={12} weight="fill" />;
+  }
+};
+
+export const EventCard = ({ event }: EventCardProps): JSX.Element => {
+  const { formatDate, formatNumber, formatMessage } = useContext(IntlContext);
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [mobile] = useMediaQuery(BREAKPOINT_MD_QUERY);
+
+  const toggleText = () => {
+    setIsExpanded(!isExpanded);
+  };
+
+  const isLocationTooLong =
+    (event.location?.length ?? 0) >= LOCATION_MAX_LENGTH;
+
+  return (
+    <Box
+      borderRadius={['xl', null, null, '2xl']}
+      height="full"
+      width="full"
+      border="1px solid var(--boemly-colors-gray-200)"
+      background="white"
+    >
+      <Box
+        position="relative"
+        width="full"
+        height={['32', null, null, '44']}
+        borderTopRadius={['xl', null, null, '2xl']}
+        css={css`
+          & span,
+          div,
+          img {
+            border-top-left-radius: inherit;
+            border-top-right-radius: inherit;
+          }
+        `}
+      >
+        <Image
+          src={strapiMediaUrl(event.image?.img, 'medium')}
+          alt={event.image?.alt}
+          fill
+          style={{
+            objectFit: event.image?.objectFit || 'cover',
+          }}
+        />
+
+        <Box
+          position="absolute"
+          top={['6', null, null, '8']}
+          right={['6', null, null, '8']}
+          zIndex="1"
+          width={['12', null, null, '16']}
+          height={['12', null, null, '16']}
+        >
+          <Image
+            src={strapiMediaUrl(event.logo.img, 'medium')}
+            alt={event.logo.alt}
+            fill
+            style={{
+              objectFit: event.logo.objectFit || 'contain',
+              borderRadius: '6px',
```

Comment:
Can you use a css variable like this?
```suggestion
              borderRadius: 'var(--boemly-radii-md)',
```